Source/Core/Core/DSP/Interpreter/DSPInterpreter.cpp

```diff
@@ -25,10 +26,10 @@ void ExecuteInstruction(const UDSPInstruction inst)
 
   if (opcode_template->extended)
   {
-    GetExtOpTemplate(inst)->intFunc(inst);
```

Comment:
Is GetExtOpTemplate(inst) still in use? I miss the removal of this old function.

Reply:
It isn't used. Like I said in the main comment, cleanup is for another PR